kernel-default.properties

```diff
@@ -454,69 +454,69 @@ mosip.role.admin.masterdata.getbiometrictypes=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN
 mosip.role.admin.masterdata.getbiometrictypeslangcode=ZONAL_ADMIN,GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER
 mosip.role.admin.masterdata.getbiometrictypescodelangcode=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN,GLOBAL_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER
 mosip.role.admin.masterdata.postbiometrictypes=ZONAL_ADMIN,GLOBAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordswords=ZONAL_ADMIN,GLOBAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.putblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.putblocklistedwordsdetails=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordswords=ZONAL_ADMIN,GLOBAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.putblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.putblocklistedwordsdetails=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
 mosip.role.admin.masterdata.deleteblocklistedwordsword=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
 mosip.role.admin.masterdata.patchblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,REGISTRATION_ADMIN
```

Comment:
Is blocklistedWord patch not invoked from UI?

Reply:
added MASTERDATA_ADMIN role to blocklistedwords patch